mleap-spring-boot/src/main/resources/mleap_serving_1.0.0_swagger.yaml

```diff
@@ -189,12 +189,28 @@ paths:
             application/x-protobuf:
               schema:
                 $ref: '#/components/schemas/DefaultLeapFrame'
+  '/actuator/health':
+    get:
+      tags:
+        - health-check
+      summary: Health check for mleap server
+      description: Health check for mleap server
+      responses:
+        '200':
+          description: successful operation
+          content:
+            application/json:
+              schema:
+               $ref: '#/components/schemas/HealthCheckResponse'
+            application/x-protobuf:
+              schema:
+                $ref: '#/components/schemas/HealthCheckResponse'
```

Comment:
There's a couple of endpoints that are enabled by default by the actuator.

```
{
    "_links": {
        "self": {
            "href": "http://localhost:8080/actuator",
            "templated": false
        },
        "health": {
            "href": "http://localhost:8080/actuator/health",
            "templated": false
        },
        "info": {
            "href": "http://localhost:8080/actuator/info",
            "templated": false
        }
    }
}
```
Instead of adding them to the swagger yaml file, could you please just add a note in the README.md file in this project, that Spring Actuator endpoints enabled by default are also available? Thank you!